src/main/java/codeu/model/store/basic/DefaultDataStore.java

```diff
@@ -99,7 +100,7 @@ private void addRandomUsers() {
 
     for (int i = 0; i < DEFAULT_USER_COUNT; i++) {
       //PART 1: Temp "password" passed to allow proper compilation 
-      User user = new User(UUID.randomUUID(), randomUsernames.get(i), "password" ,Instant.now());
```

Comment:
Spacing issue?

Reply:
Oh yeah, oops, I'll fix that

Reply:
The passwords from part 1 and 2 were changed to password Hashes in part 8/9

Reply:
Yeah. I noticed that as well.